python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -264,12 +264,12 @@ def fromDict(cls, dictionary):
 
         calibVersion = dictionary['metadata']['bfk_VERSION']
         if calibVersion == 1.0:
-            calib.log.warning("Old Version of brighter-fatter kernel found. Current version: "
-                              f"{calib._VERSION}. The new attribute 'expIdMask' will be "
-                              "populated with 'True' values, and the new attributes 'rawMeans'"
-                              "and 'rawVariances' will be populated with the masked 'means'."
-                              "and 'variances' values."
-                              )
+            calib.log.info("Old Version of brighter-fatter kernel found. Current version: "
+                           f"{calib._VERSION}. The new attribute 'expIdMask' will be "
+                           "populated with 'True' values, and the new attributes 'rawMeans'"
+                           "and 'rawVariances' will be populated with the masked 'means'."
```

Comment:
There's an extra `.` at the end of this line.

Reply:
Fixed, as well as added the missing spaces.